docker-hub/scim.md

```diff
@@ -1,42 +1,44 @@
 
-We currently support the following provisioning features: creating new users, pushing user profile updates, removing users, deactivating users and reactivating users.
+We currently support the following provisioning features: creating new users, push user profile updates, remove users, deactivate users and reactivate users.  Group management is currently not supported.
 
-## Configure
+The table below lists the supported attributes.
 
-SAML-based SSO must be properly set up and functional for your organization before you start configuring automated provisioning. In addition, you must verify your company domain and have the appropriate connectors in your IdP. Your user email domain must be the same company domain you use for Single Sign-on (SSO). Enforcing SSO is not required to enable SCIM. However, you must [configure SAML SSO](../single-sign-on/index.md){: target="_blank" rel="noopener" class="_"} before you enable SCIM.
+| Attribute    | Description
+|:---------------------------------------------------------------|:-------------------------------------------------------------------------------------------|
+| username             | Unique identifier of the user (email)                                   |
+| givenName                            | User’s first name |
+| familyName |User’s surname                                              |
+
+## Configure
 
-### Okta IdP
+SAML-based SSO must be properly set up and be functional for your organization before you start configuring automated provisioning. In addition, you must verify your company domain and have the appropriate connectors in your IdP. Your user email domain must be the same company domain you use for Single Sign-on (SSO). Enforcing SSO is not required to enable SCIM. However, you must [configure SAML SSO](../single-sign-on/index.md){: target="_blank" rel="noopener" class="_"} before you enable SCIM.
 
-Before making SCIM configuration changes in your IdP, navigate to [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} and select **Organizations** > **Settings** > **Security**. SCIM is locked until you complete the SSO configuration and verify your company domain. Enable SCIM and access your **Base URL** and **API Token** ( the same as Bearer Token in Okta).
+Before you make SCIM configuration changes in your IdP, navigate to [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} and select **Organizations** > **Settings** > **Security**. SCIM is locked until you complete the SSO configuration and verify your company domain. Enable SCIM and access your Base URL and API Token. You can also generate a new API token.
 
 ![scim-provisioning](images/scim-provisioning.png){:width="700px"}
```

Comment:
suggestion: Update this image as well as it shows an invalid SCIM Base URL can could lead to confusions